### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by febs** [Fightingbox/Fightingbox-mini#5]
This controller is really beautiful.

Is there any chance for it to see an update based on the mainstream code?

Thank you,

**Comment by manksalot** [comment]
It's possible to update to the latest GP2040-CE firmware. [Flash nuke](https://gp2040-ce.info/installation/#flash-nuke-process) the device and then install the [latest release](https://github.com/OpenStickCommunity/GP2040-CE/releases) that ends in FightboardV3.uf2

**Comment by shinriyo** [comment]
It it compatible?
I use FIGHTINGBOX MINI which bought in AliErpress.
It can't detect one rotation command. So I want to update.

**Comment by shinriyo** [comment]
I tried the latest version that was on this GitHub and the one you linked to, but I can't get it to start.

**Comment by manksalot** [comment]
Yes, I also have a FB mini from aliexpress. I followed the GP2040-CE instructions to flash nuke first and then I was able to successfully update to GP2040-CE_0.7.10_FightboardV3.uf2 
![Imagepipe_1](https://github.com/user-attachments/assets/74ce9281-435b-4e5f-be1e-53f6b9ad0872)

**Comment by shinriyo** [comment]
@manksalot
Thank you for replying.
I downloaded `GP2040-CE_0.7.10_FightboardV3.uf2` and just put it again. It was Successful!
<img width="523" alt="Screenshot 2024-11-24 at 20 49 49" src="https://github.com/user-attachments/assets/040d1c35-f3ee-4200-acc3-ac8dd1b5ff4a">

**Comment by shinriyo** [comment]
Excellent! The `GP2040-CE_0.7.10_FightboardV3.uf2` also detects a single rotation command.
I makes Zangief screw pile driver!

**Comment by febs** [comment]
@shinriyo @manksalot I've done that but apparently the buttons in the LCD screen are all wrong. Did you manage to put the correct layout in there?

**Comment by febs** [comment]
I updated it, so you can have the proper thing.
https://github.com/febs/Fightingbox-mini/releases/tag/v0.7.10

**Comment by shinriyo** [comment]
https://github.com/OpenStickCommunity/GP2040-CE/releases/tag/v0.7.11
Why ther are not `GP2040-CE_0.7.11_FightboardV3.uf2`?
